ci-operator/step-registry/osd-ccs/cluster/provision/gcp/osd-ccs-cluster-provision-gcp-commands.sh

```diff
@@ -88,9 +181,15 @@ echo "  Channel group: ${CHANNEL_GROUP}"
 echo "  Etcd encryption: ${ETCD_ENCRYPTION}"
 echo "  Disable workload monitoring: ${DISABLE_WORKLOAD_MONITORING}"
 echo "  Subscription type: ${SUBSCRIPTION_TYPE}"
+if [[ -v IS_XPN ]]; then
+  echo "$(date -u --rfc-3339=seconds) - INFO: Install into GCP Shared VPC"
```

Comment:
I do not think we need to treat this as a special. Just printing the parameters is enough.